core/src/HIP/Kokkos_HIP_Instance.hpp

```diff
@@ -123,6 +124,11 @@ class HIPInternal {
 
   bool was_finalized = false;
 
+  // FIXME_HIP: these want to be per-device, not per-stream...  use of 'static'
+  // here will break once there are multiple devices though
+  static unsigned long *constantMemHostStaging;
+  static hipEvent_t constantMemReusable;
```

Comment:
Can we call this something that indicates it's an event when read in code? Like `constantMemAvailableEvent` or something?